common/src/main/java/com/wynntils/wynn/objects/profiles/ToolProfile.java

```diff
@@ -7,14 +7,7 @@
 import com.wynntils.wynn.objects.profiles.ingredient.ProfessionType;
 import java.util.Locale;
 
-public class ToolProfile {
-    private final ToolType toolType;
-    private final int tier;
-
-    public ToolProfile(ToolType toolType, int tier) {
-        this.toolType = toolType;
-        this.tier = tier;
-    }
+public record ToolProfile(com.wynntils.wynn.objects.profiles.ToolProfile.ToolType toolType, int tier) {
```

Comment:
Did you just convert a class that is meant to be parsed with gson into a record? That won't work. (where is the data from?)

Reply:
I agree that that won't work. And that is not what I am doing. The "ToolProfile" is perhaps not entirely named according to our standards. That depends on if you interpret the `-Profile` as *how the class is used* vs *where we get the data from*. It aligns with the former, but not the latter. The Profile records are created by the `ToolAnnotator` class when a tool item is parsed.

Reply:
Oh. Can we reserve the "*Profile" names for API/GSON deserialization (or alternatively come up with a some other name for it)?

Reply:
Yeah... I've been thinking a bit about different names, but not found any good ones. I think "instance" is a good name for a specific item of some kind, but I keep coming back to "class" or "type" for the "general" concept, and it is so vague. "Profile" is rather odd, but it has become a bit of our nomenclature. Synonym search gives "sort", "variety", "kind" as possible alternatives. 

Hm. Maybe "Kind" would be okay? Like in "ToolKind"?

Reply:
@kristofbolyai Is this question on naming a blocker for this PR?